config/initializers/letter_opener.rb

```diff
@@ -0,0 +1,5 @@
+if Object.const_defined? :LetterOpener
+  LetterOpener.configure do |config|
```

Comment:
[dust] I'd be a bit more comfortable with this in `config/environments/development.rb` so it couldn't run in unexpected evs